Android/ARCore/source/Google.ARCore.csproj

```diff
@@ -2,8 +2,7 @@
 <Project DefaultTargets="Build" ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
     <AndroidCodegenTarget>XAJavaInterop1</AndroidCodegenTarget>
-    <TargetFrameworkVersion>v9.0</TargetFrameworkVersion>
-    <ReleaseVersion>1.14.0</ReleaseVersion>

```

Comment:
Should this be removed?

Reply:
@JonDouglas Seems that cake script or IDE removes it. This is 2nd time that happened and I did not do it on purpose (manually).

Reply:
restored